samples/apps/copilot-chat-app/webapi/Skills/ChatMessage.cs

```diff
@@ -19,6 +19,7 @@ public enum AuthorRoles
     {
         User = 0,
         Bot,
+        Participant
```

Comment:
What is the participant in this context? Maybe its a good time to add a comment describing each role.